core/common/src/test/java/alluxio/AlluxioTestDirectory.java

```diff
@@ -45,11 +44,10 @@
    */
   public static File createTemporaryDirectory(String prefix) {
     final File file = new File(ALLUXIO_TEST_DIRECTORY, prefix + "-" + UUID.randomUUID());
-    try {
-      file.createNewFile();
```

Comment:
before this had only one usage, and that usage was just to reserve a temporary name. We would delete the result and then create an actual directory.